packages/frontend/src/app/dataContracts/page.js

```diff
@@ -12,7 +12,10 @@ export const metadata = {
   applicationName: 'Dash Platform Explorer'
 }
 
-function DataContractsRoute () {
+function DataContractsRoute ({ searchParams }) {
+  const page = Number(searchParams.p) || 1
```

Comment:
Why p?
Just page, nuh?

Reply:
I think using "p" will make the URL shorter and more concise.